pkg/controller/vsphere/session/session.go

```diff
@@ -67,6 +67,7 @@ func GetOrCreate(
 			return &session, nil
 		}
 	}
+	klog.Infof("Create new vcenter session")
```

Comment:
Is this definitely something we want? If so, I'd suggest
```suggestion
	klog.Infof("No existing vCenter session found, creating new session")
```

Reply:
This should appear just once during controller startup. IMHO would be nice to have such message.

Updated message